app/src/main/kotlin/dev/patrickgold/florisboard/lib/snygg/value/SnyggValue.kt

```diff
@@ -105,8 +107,12 @@ object SnyggImplicitInheritValue : SnyggValue, SnyggValueEncoder {
     override fun encoder() = this
 }
 
-val SnyggVarValueEncoders = listOf(
+val isSOrAbove = Build.VERSION.SDK_INT >= Build.VERSION_CODES.S
+
+val SnyggVarValueEncoders = listOfNotNull(
     SnyggSolidColorValue,
+    if (isSOrAbove) SnyggMaterialYouLightColorValue else null,
+    if (isSOrAbove) SnyggMaterialYouDarkColorValue else null,
```

Comment:
For version checking we can use AndroidVersion:

```suggestion
val SnyggVarValueEncoders = listOfNotNull(
    SnyggSolidColorValue,
    if (AndroidVersion.ATLEAST_API31_S) SnyggMaterialYouLightColorValue else null,
    if (AndroidVersion.ATLEAST_API31_S) SnyggMaterialYouDarkColorValue else null,
```